driver/driver.c

```diff
@@ -103,8 +103,7 @@ DriverEntry(PDRIVER_OBJECT DriverObject,
     DriverObject->MajorFunction[IRP_MJ_PNP] = 0 != StorPortDispatchPnp ? WnbdDispatchPnp : 0;
     GlobalExt = NULL;
 
-    WNBD_LOG_LOUD(": Exit");
-
+    WNBD_LOG_LOUD("Exit");
```

Comment:
Using one commit per file is weird and pollutes the git log, let's squash them.

Reply:
It's easier to see the changes in my opinion, but sure I will squash them in one commit.